File: SystemAPI/DTO/UsuarioDTO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace SystemAPI.DTO
{
    public class UsuarioDTO
    {
        public string Nome { get; set; }
        public string Senha { get; set; }

        public UsuarioDTO(string nome, string senha)
        {
            this.Nome = nome;
            this.Senha = senha;
        }
        public UsuarioDTO()
        {

        }
    }
}

File: Front/Cliente/helpclient/src/Modal/UsuarioDTO.ts
import React from "react";
export class UsuarioDTO{
    nome: string
    pass: string

    
    public get Pass() : string {
        return this.pass
    }

    
    public get Nome() : string {
        return this.nome
    }
    
    
    constructor({nome, pass}:{nome: string, pass: string}){
        this.nome = nome
        this.pass = pass
    }
}